pkgs/tools/text/snippetpixie/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  vala,
+  pkgconfig,
+  wrapGAppsHook,
+  cmake,
+  appstream,
+  desktop-file-utils,
+  python3,
+  libgee,
+  gtk3,
+  sqlite,
+  at-spi2-atk,
+  at-spi2-core,
+  dbus,
+  ibus,
+  json-glib,
+  pantheon
```

Comment:
your comma's are backwards here for long attribute sets.

> https://nixos.org/nixpkgs/manual/#chap-conventions
>  but if they don't fit on one line they're written as: 

Example is out of date but it's like in haskell
```
{ stdenv
, something
, something
, something
, ... 
```